dbms/src/Storages/Transaction/RegionTable.h

```diff
@@ -179,6 +218,8 @@ class RegionTable : private boost::noncopyable
 private:
     TableMap tables;
     RegionInfoMap regions;
+    SafeTsMap leader_safets;
+    SafeTsMap self_safets;
```

Comment:
```suggestion
    SafeTsMap leader_safe_ts;
    SafeTsMap self_safe_ts;
```

Reply:
Maybe it is more reasonable to make `leader_safe_ts` and `self_safe_ts` to be vars of class `Region`? So that we won't run into we store the safe ts without a Region instance.

Reply:
> Maybe it is more reasonable to make `leader_safe_ts` and `self_safe_ts` to be vars of class `Region`? So that we won't run into we store the safe ts without a Region instance.

But `leader_safe_ts ` and `self_safe_ts` are the maps for <region_id, safe_ts>, so they should be in `RegionTable` class.

Reply:
Can we make `UInt64 leader_safe_ts` and `UInt64 self_safe_ts` as the member of `Region`?